pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{lib, stdenv, fetchFromGitHub, bash, fakeroot, libinput, xdotool, makeWrapper}:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  buildInputs = [ bash fakeroot makeWrapper ];
+
+  installPhase =
+    ''
+      mkdir -p $out
+      fakeroot -- bash libinput-gestures-setup -d "$out" install
+      mv $out/usr/bin $out/bin-unwrapped
+      mv $out/usr/share $out/share
```

Comment:
```fixupPhase``` will handle this for you, you don't need to do it manually

Reply:
That doesn't seem to happen for me, unless I'm missing something.